modules/mcagar/src/main/ts/ephox/mcagar/api/pipeline/TinyScenarios.ts

```diff
@@ -5,9 +5,18 @@ import * as fc from 'fast-check';
 import { Editor } from '../../alien/EditorTypes';
 
 type ContentGenerator = fc.Arbitrary<SugarElement<HTMLElement>>;
-interface SelectionExclusions { containers: (container: SugarElement<Node>) => boolean }
-interface ArbScenarioOptions { exclusions: SelectionExclusions }
-interface AsyncPropertyOptions { scenario: ArbScenarioOptions; property: fc.Parameters }
+
+interface SelectionExclusions {
+  readonly containers: (container: SugarElement<Node>) => boolean;
+}
+
+interface ArbScenarioOptions {
+  readonly exclusions: SelectionExclusions;
+}
+
+interface AsyncPropertyOptions {
+  readonly scenario: ArbScenarioOptions; property: fc.Parameters;
```

Comment:
This seems weird. Why are they on the same line?

Reply:
This is just what the autofixer did and we don't currently have any other linter rule to enforce them being on multiple lines so it happily left them there. I'll fix this and the issue Spocke mentioned.